otel-loadbalancer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build the target allocator binary
```

Comment:
Looks like this got duplicated.  Ensure to `git rm -r` the old directory.  Maybe also squash the commits on this branch before making a PR upstream.